src/module_cache/WasmModuleCache.cpp

```diff
@@ -24,75 +24,26 @@ int WasmModuleCache::getCachedModuleCount(const std::string& key)
     return count;
 }
 
-std::string WasmModuleCache::getBaseCachedModuleKey(const faabric::Message& msg)
-{
-    std::string key = msg.user() + "/" + msg.function();
-    return key;
-}
-
-/**
- * There are two kinds of cached module here, the "base" cached module, i.e. the
- * default module with its zygote function executed, (same for all instances),
- * or one of many "special" cached modules, those restored from snapshots
- * captured at arbitrary points (e.g. when spawning a thread).
- */
 wasm::WAVMWasmModule& WasmModuleCache::getCachedModule(
   const faabric::Message& msg)
 {
-    const std::shared_ptr<spdlog::logger>& logger = faabric::util::getLogger();
+    auto logger = faabric::util::getLogger();
 
-    // Get the keys for both types of cached module
-    const std::string baseKey = getBaseCachedModuleKey(msg);
+    std::string baseKey = faabric::util::funcToString(msg, false);
 
-    // Check for the base cached module (required for restoring from snapshot as
-    // well)
+    // Check for the base cached module (must be present for restoring a
+    // snapshot too)
     if (getCachedModuleCount(baseKey) == 0) {
         faabric::util::FullLock lock(mx);
         if (cachedModuleMap.count(baseKey) == 0) {
             // Instantiate the base module
             logger->debug("Creating new base zygote: {}", baseKey);
             wasm::WAVMWasmModule& module = cachedModuleMap[baseKey];
             module.bindToFunction(msg);
-
```

Comment:
The `WasmModuleCache` was aware of snapshot and restore previously, which seemed odd as its purpose is really just to cache the base cached version of each wasm module. Now it only ever holds a single cached module for each function.